Resources/doc/installation.rst

```diff
@@ -78,6 +78,20 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+Authorization
+-------------
+
+If you use authorization on your MongoDB database you can provide username, password, and authentication database in the following way:
+
+    # app/config/parameters.yml
+    parameters:
+        mongodb_server: "mongodb://username:password@localhost:27017/auth-db"
```

Comment:
The [authSource](https://docs.mongodb.com/manual/reference/connection-string/#urioption.authSource) URI option is the preferred mechanism for specifying the authentication database. I'd suggest replacing `/auth-db` with `/?authSource=auth-db` here.

The `/auth-db` would actually specify both a default database _and_ default authentication database. Most drivers (including PHP) have no concept of a default database, since their API typically requires users to explicitly select a database and collection through the client object. Doctrine ODM does have a default database (when unspecified in model mappings), but I don't recall if it parses it out of the connection string (perhaps @malarzm can confirm). In any event, `authSource` is more explicit and also avoids any ambiguity in the event that Doctrine _does_ parse the URI and a user might need to provide distinct values for both.

I'll also point out that for some auth mechanisms (e.g. LDAP, Kerberos), the value of `authSource` does not represent a database name at all, so that is another reason not to rely on the overloaded `/auth-db` component in modern applications.

See [Connection String: Components](https://docs.mongodb.com/manual/reference/connection-string/#components) in the MongoDB manual for more information.

Reply:
Thanks for the background. I tested the connection with the url param notation and it works.